test/Microsoft.TestPlatform.TestUtilities/IntegrationTestBase.cs

```diff
@@ -379,7 +378,8 @@ public virtual string GetConsoleRunnerPath()
             }
```

Comment:
Variable name are bit confusing.
When will this isVsTestConsoleDLLScenario be different from IsNetCoreRunner() ?

Doubt arising because RunTestsWithFilterTests invokes GetConsoleRunnerPath( IsNetCoreRunner())